### PR TITLE
Add EIP: Public Non-Fungible Tokens Emote Repository

### PR DESCRIPTION
**Pull request by ThunderDeliverer**
This proposal fixes universal and forward compatibility with Unicode emotes of ERC-6831.

### CODE REVIEW

File: EIPS/eip-7381.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
+---
+eip: 7381
```
Comment (SamWilsn) [inline EIPS/eip-7381.md:2]:
```suggestion
eip: 7409
```

Comment (ThunderDeliverer) [reply EIPS/eip-7381.md]: We needed to update the assets and the whole proposal as well, so I did it in the editor.

File: EIPS/eip-7409.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+category: ERC
+created: 2023-07-26
+requires: 165
+---
+
+> :information_source: **[ERC-7409](./eip-7409.md) has superseded [ERC-6381](./eip-6381.md).** :information_source:  
+> [ERC-7409](./eip-7409.md) fixes the compatibility issue in the [ERC-6381](./eip-6381.md) interface specification, where the UNIX emotjis are represented using `bytes4` values. The introduction of variation flags and emoji skin tones has rendered the `bytes4` namespace insufficient for representing all possible emojis, so the new standard used `string` instead.
+> Apart from this fix, [ERC-7409](./eip-7409.md) is functionally equivalent to [ERC-6381](./eip-6381.md).
+>
+> :warning: [ERC-7409](./eip-7409.md) MUST be used in lieu of [ERC-6381](./eip-6381.md). :warning:
```
Comment (SamWilsn) [inline EIPS/eip-7409.md:18]:
```suggestion
```

This information should fit in the motivation/abstract for the most part.

File: EIPS/eip-7409.md [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+
+## Abstract
+
```
Comment (SamWilsn) [inline EIPS/eip-7409.md:21]:
```suggestion

:information_source: **[ERC-7409](./eip-7409.md) supersedes [ERC-6381](./eip-6381.md).** :information_source:
```

File: EIPS/eip-7409.md [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+## Motivation
+
+With NFTs being a widespread form of tokens in the Ethereum ecosystem and being used for a variety of use cases, it is time to standardize additional utility for them. Having the ability for anyone to interact with an NFT introduces an interactive aspect to owning an NFT and unlocks feedback-based NFT mechanics.
+
+This ERC introduces new utilities for [ERC-721](./eip-721.md) based tokens in the following areas:
+
+- [Interactivity](#interactivity)
+- [Feedback based evolution](#feedback-based-evolution)
+- [Valuation](#valuation)
+
```
Comment (SamWilsn) [inline EIPS/eip-7409.md:35]:
```suggestion

This proposal fixes the compatibility issue in the [ERC-6381](./eip-6381.md) interface specification, where emojis are represented using `bytes4` values. The introduction of variation flags and emoji skin tones has rendered the `bytes4` namespace insufficient for representing all possible emojis, so the new standard used `string` instead. Apart from this fix, this proposal is functionally equivalent to [ERC-6381](./eip-6381.md).
```

File: EIPS/eip-7409.md [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+This proposal fixes the compatibility issue in the [ERC-6381](./eip-6381.md) interface specification, where emojis are represented using `bytes4` values. The introduction of variation flags and emoji skin tones has rendered the `bytes4` namespace insufficient for representing all possible emojis, so the new standard used `string` instead. Apart from this fix, this proposal is functionally equivalent to [ERC-6381](./eip-6381.md).
+### Interactivity
```
Comment (ThunderDeliverer) [inline EIPS/eip-7409.md:33]:
```suggestion

### Interactivity
```

File: EIPS/eip-7409.md [hunk 6]
```diff
@@ -0,0 +1,376 @@
...
+category: ERC
+created: 2023-07-26
+requires: 165
+---
+
+
+## Abstract
+
+:information_source: **[ERC-7409](./eip-7409.md) supersedes [ERC-6381](./eip-6381.md).** :information_source:
+The Public Non-Fungible Tokens Emote Repository standard provides an enhanced interactive utility for [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) by allowing NFTs to be emoted at.
```
Comment (SamWilsn) [inline EIPS/eip-7409.md:18]: Same as on the other one, but `information_source` doesn't render to anything. Maybe use an emoji?

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**